dashboard/db/schema.rb

```diff
@@ -787,6 +787,13 @@
     t.index ["parent_level_id"], name: "index_parent_levels_child_levels_on_parent_level_id"
   end
 
+  create_table "parental_permission_requests", charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.integer "user_id", null: false
```

Comment:
I think this needs to be updated as well (based on the change to the migration). It looks like Drone is failing due to it not liking the schema right now. 

I think if you run `rails db:migrate` again locally, this schema should get updated appropriately.

Reply:
I ran `rails db:migrate` again, but I think the merge conflict is due to your LTI Integration table being merged. Let me try rebasing past that and then re-generating the schemas.